Dockerfile

```diff
@@ -19,7 +19,7 @@
 # To `docker run` with your locally built image, replace `umaprotocol/protocol` with <username>/<imagename>.
 
 # Fix node version due to high potential for incompatibilities.
-FROM node:lts
+FROM node:14
```

Comment:
Does this not break a lot of assumptions?

Reply:
Also can you comment specifically why we want 14, specifcally we want the `?` syntax

Reply:
Pretty sure it doesn't break anything. I've tested it quite thoroughly and haven't noticed any issues with it. Specifically, I've tried out the voter dapp & liquidator/disputer/monitor tests with no issues. 

Are there other things you'd like me to test?